content/applications/sales/point_of_sale/overview/getting_started.rst

```diff
@@ -23,44 +24,56 @@ Configure payment methods
 -------------------------
 
 To add a payment method, you first need to create it. Go to :menuselection:`Point of Sale -->
-Configuration --> Payment Methods --> Create`. Set a name and select the **payment terminal** or
-check :guilabel:`Cash` for cash payments.
+Configuration --> Payment Methods --> Create`. Set a name and select the :guilabel:`Journal`. Select
+:guilabel:`Cash` to use this payment method for cash payments, or :guilabel:`Bank` to use it for
+card payments.
+
+Check :guilabel:`Identify Customer` to allow this payment method *exclusively* for identified
+customers. For any unidentified customers to be able to pay, leave the :guilabel:`Identify Customer`
```

Comment:
- Not sure to understand what "identified customers" refer to. 
- I would probably move this part before the "Journal" information as the "Identify Customer" field is above the Journal one.